### PR TITLE
Design with CSS Grid

### PR DESCRIPTION
**Pull request by jeymarsonwolox**
Summary
Change of design to css grid

Screenshots
captura de pantalla 2018-11-02 a la s 3 00 59 p m
captura de pantalla 2018-11-02 a la s 3 01 22 p m
captura de pantalla 2018-11-02 a la s 3 01 34 p m

Trello Card
https://trello.com/c/KOZDpaqq/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <html lang="en">
+
```
Comment (FrankIglesias) [inline index.html]: delete this lines

File: index.html [hunk 2]
```diff
@@ -8,15 +9,16 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
```
Comment (FrankIglesias) [inline index.html]: same

File: index.html [hunk 3]
```diff
@@ -8,15 +9,16 @@
...
   <nav class="nav">
     <h3 class="nav-title">
       Developer Tools
     </h3>
   </nav>
-  <section class="main">
+  <section class="container">
     <h2 class="title">Front-End Tools</h2>
-    <div class="tech">
+    <div class="tech tech-fr tech-pr">
```
Comment (FrankIglesias) [inline index.html]: why this suffixes?

**Review by FrankIglesias — APPROVED** [review]

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
Check the comments

**Comment by jeymarsonwolox** [comment]
Corrections are made to annotations of the pull request.

File: scss/tech.scss [hunk 4]
```diff
@@ -6,35 +6,32 @@ $height: 312px;
 $width: 218px;
 $border-radius: 5px;
 
+%tech-pr {
+  border-right: solid 2px $green;
+  height: 100%;
+  margin-left: 50%;
+  padding: 50% 10% 50% 0;
```
Comment (FrankIglesias) [inline scss/tech.scss]: use px instead for this props

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by jeymarsonwolox** [comment]
Correction of props to px

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**